### PR TITLE
make multicore default to false

### PR DESCRIPTION
**Pull request by chicco785**

### CODE REVIEW

File: charts/iot-agent/README.MD [hunk 1]
```diff
@@ -53,6 +53,8 @@ chart and their default values.
 | `priorityClassName`               | If specified, indicates the pod's priority  | ``                                                  |
 | `affinity`               | If specified, defines affinity rules  | ``                                                  |
 
+Be careful with enabling `multiCore` on k8s running on top of aws nodes.
```
Comment (c0c0n3) [inline charts/iot-agent/README.MD:56]: perhaps we should explain why or at least what to watch out for?

**Review by c0c0n3 — COMMENTED** [review]
What was the problem with multicore on?

File: charts/iot-agent/README.MD (hunk 1 above)
Comment (chicco785) [reply charts/iot-agent/README.MD:56]: processors a constantly spawn and killed, and the service is not working...

Comment (c0c0n3) [reply charts/iot-agent/README.MD]: holy moly!

**Review by c0c0n3 — APPROVED** [review]

**Pull request merged**